DEVELOPING.rst

```diff
@@ -8,461 +8,91 @@
-  would typically start your topical branch from the ``production``
-  branch;
-
-- for new developments, you would typically start your topical branch
-  from the ``master`` branch.
-
-Here is example:
-
-.. code-block:: console
-
-   $ git checkout master
-   $ git checkout -b improve-event-display-icons
-   $ $EDITOR some_file.py
-   $ git commit -a -m 'some improvement'
-   $ $EDITOR some_other_file.py
-   $ git commit -a -m 'some other improvement'
-
-When everything is ready, you may want to rebase your topical branch
-to get rid of unnecessary commits:
+4. If you want to load only some records/docs
 
 .. code-block:: console
 
-   $ git checkout improve-event-display-icons
-   $ git rebase master -i # squash commits here
-
-Making pull requests
---------------------
-
-You are now ready to issue a pull request: just push your branch in
-your personal repository:
-
-.. code-block:: console
-
-   $ git push origin improve-event-display-icons
-
-and use GitHub's "Pull request" button to make the pull request.
-
-Watch GitHub Actions build status report to see whether your pull request
-is OK or whether there are some troubles.
-
-Updating pull requests
-----------------------
-
-Consider the integrator had some remarks about your branch and you
-have to update your pull request.
-
-Firstly, update to latest upstream "master" branch, in case it may
-have changed in the meantime:
-
-.. code-block:: console
-
-   $ git checkout master
-   $ git fetch upstream
-   $ git merge upstream/master --ff-only
-
-Secondly, make any required changes on your topical branch:
-
-.. code-block:: console
-
-   $ git checkout improve-event-display-icons
-   $ $EDITOR some_file.py
-   $ git commit -a -m 'amends something'
-
-Thirdly, when done, interactively rebase your topical branch into
-nicely organised commits:
-
-.. code-block:: console
-
-   $ git rebase master -i # squash commits here
-
-Finally, re-push your topical branch with a force option in order to
-update your pull request:
-
-.. code-block:: console
-
-   $ git push origin improve-event-display-icons -f
-
-Finishing pull requests
------------------------
-
-If your pull request has been merged upstream, you should update your
-local sources:
-
-.. code-block:: console
-
-   $ git checkout master
-   $ git fetch upstream
-   $ git merge upstream/master --ff-only
-
-You can now delete your topical branch locally:
-
-.. code-block:: console
+   $ my_docker exec -it web cernopendata fixtures records --file /content/data/records/<full_path>
+   $ my_docker exec -it web cernopendata fixtures docs --file /content/data/docs/<full_path>
 
-   $ git branch -d improve-event-display-icons
+..
 
-and remove it from your repository as well:
+5. Finally, if there are new images, ensure that they appear in the correct folder
 
 .. code-block:: console
 
-   $ git push origin master
-   $ git push origin :improve-event-display-icons
+   $ my_docker exec -it web /data/scripts/load-images.sh
```

Comment:
done, thanks

Reply:
I tested that the images are well copied, for example:

```
[invenio@d2d5c9b658be code]$ ls -l /content/data/images/subdir/logo-iris-hep.png
-rw-r--r-- 1 invenio 1000 12901 Aug 13 07:21 /content/data/images/subdir/logo-iris-hep.png
```

However the existing content images are still located in the traditional place:

```
[invenio@d2d5c9b658be code]$ ls -l /code/cernopendata/modules/theme/static/docs/cms-releases-heavy-ion-data/cms_hi_event_1.png
-rw-r--r-- 1 invenio 1000 288993 Feb 27 17:36 /code/cernopendata/modules/theme/static/docs/cms-releases-heavy-ion-data/cms_hi_event_1.png
```

Two questions:

1. How are the copied user-supplied images exposed over the web? Under which URL?
2. It would be good to keep all the existing content images such as `cms_hi_event_1.png` here in the new directory as well, as suggested above. In this way, the users would have complete example how to add a Markdown docs with embedded images in one place.